src/mlpack/methods/regularized_svd/regularized_svd_function.cpp

```diff
@@ -129,8 +129,8 @@ void RegularizedSVDFunction::Gradient(const arma::mat& parameters,
 namespace mlpack {
 namespace optimization {
 
-template<>
-double StandardSGD<mlpack::svd::RegularizedSVDFunction>::Optimize(
+template <> template <>
```

Comment:
I think we should put these on two separate lines.  Also, I notice a lot of `template <typename ...>`, but I think we should use `template<typename ...>` (i.e. no space).  I'm not sure why the style check doesn't pick that up, I haven't looked into it.